frontend/public/texts/profile_texts.js

```diff
@@ -108,16 +108,16 @@ export default function getProfileTexts({ profile, locale }) {
       de: ", um die vollständigen Informationen dieses/dieser Benutzer*in zu sehen",
     },
     your_projects: {
-      en: "Your Climate Projects",
-      de: "Deine Klimaschutzprojekte",
+      en: "Your projects",
+      de: "Deine Projekte",
     },
     this_users_projects: {
       en: `${profile?.first_name}'s Climate Projects`,
       de: `Klimaschutzprojekte von ${profile?.first_name}`,
     },
     your_ideas: {
-      en: "Your Climate Ideas",
-      de: "Deine Klimaschutzideen",
+      en: "Your ideas",
+      de: "Deine Organisationen",
```

Comment:
Made this consistent wording across projects, orgs, and ideas:

<img width="561" alt="Screen Shot 2021-07-07 at 7 27 16 PM" src="https://user-images.githubusercontent.com/1530684/124852597-5ec53d00-df59-11eb-9b5d-005e5879b94d.png">

Not sure if I got the 🇩🇪 right!

Reply:
Actually I changed that to "climate projects" and "climate ideas" on purpose so that people who find a profile through Google can see that the platform is about climate projects and climate ideas. The reason why I didn't change it for organizations was because not every organization that has a climate projects would be considered a climate organization, especially for-profit companies.

Reply:
Ah gotcha - ok will back that out 👍